DigitalLearningSolutions.Data.Migrations/202011101344_CompetencyFrameworkDBChanges.cs

```diff
@@ -0,0 +1,172 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+    using FluentMigrator.SqlServer;
+    [Migration(202011101344)]
+    public class CompetencyFrameworkDBChanges : Migration
```

Comment:
Did you mean to include these changes? Seems unrelated to layout and error pages? 🙂